### ISSUE TITLE
wk3 june_epic: Sim UI - Responsive Design from Top Navigation Buttons blocks the Introductory Text

### ISSUE THREAD

**Issue by lzim** [lzim/teampsd#2054]
On  the SIM UI home page the log off button is appearing in the upper right (forced from the top row of icons) which is covering the introductory text.
![image](https://user-images.githubusercontent.com/59668647/121751349-c73c0e00-cac2-11eb-8947-e235d82db06f.png)

**Comment by jamesmrollins** [comment]
Hi @lijenn @dlkibbe @clairecollie  

YES - you are correct.  The buttons wraps as a part of responsive design.  If you have a smaller screen and would like all the buttons in a single row, adjust your zoom.  It will make window contents smaller.

**Comment by lijenn** [comment]
@jamesmrollins 
Update: Looks like the introductory text is blocked from the nav bars being responsive to the design as the windows minimize on both Edge or Chrome.

![image](https://user-images.githubusercontent.com/59668647/122083339-a3741300-cdb5-11eb-8fde-13fdf92a5a63.png)

I also provided an extreme situation of  the minimized window to see how much of the text gets cut off, it seems like it's just the first sentence of the intro text.
![image](https://user-images.githubusercontent.com/59668647/122083816-0f567b80-cdb6-11eb-9b09-339dc96554b4.png)

**Comment by jamesmrollins** [comment]
Hi @lijenn.  Thanks for bringing this to our attention.  In our effort to provide responsive design, it is important for us to know what screen resolution you are set at and what zoom level.  Can you update this thread with that information?

**Comment by lijenn** [comment]
Thanks @jamesmrollins!

VA GFE Laptop
- screen resolution: 1920 x 1080
- zoom level: 100%

I see you mentioning the zoom level to get the best fit, but what if a user needs to minimize the windows in order to look multiple resources and has to share their screen? (Scenario: A Facilitator or Learner only has 1 monitor and need to look at/had multiple window browsers open at once to refer to during the Live team meeting while they are screen-sharing)

My concern is that while adjusting the zoom to a really low percentage shows the full intro text, the overall screen looks too small for the average user to use/read from. 

- For example, here is a minimized window on the VA Laptop with 100% zoom level that blocks off the intro text:
![image](https://user-images.githubusercontent.com/59668647/122092842-3f564c80-cdbf-11eb-91ae-c391fad3edef.png)

- Here is after I adjust the zoom level to be 75% to show the first sentence of the intro text:
![image](https://user-images.githubusercontent.com/59668647/122093289-c60b2980-cdbf-11eb-87eb-d1417e0ee5d0.png)

Any chance it could it be the top portion that comes after the top navigation header of the sim that is not responsive? 

If it can't be responsive due to engineering restrictions, what about minimizing the white space in the header from the top nav bar to show the intro text or in the following screenshot, the blue headers?
![image](https://user-images.githubusercontent.com/59668647/122096830-d3c2ae00-cdc3-11eb-9cd5-2d75b93f8e3d.png)

Relatedly, same thing happens in the Play session page with the blue headers getting cut off when the window is minimized at 100% zoom:
![image](https://user-images.githubusercontent.com/59668647/122091918-2dc07500-cdbe-11eb-987b-e46876375017.png)

**Comment by jamesmrollins** [comment]
Hi @lijenn.  I reviewed this behavior with @hirenp-waferwire.  He indicated that we need to click the "refresh" button after we resize the window.  That way the browser can sense what area we have sized to, and then adjust view accordingly.  Then we can use the zoom, or drag the window for final adjustments for optimal viewing.  I will create some instructions regarding this for the mlt facilitator and technical manual. I will be happy to review with you tomorrow.  thanks, James
